lib/argument_parser.js

```diff
@@ -197,7 +198,9 @@ ArgumentParser.prototype.addSubparsers = function (options) {
 };
 
 ArgumentParser.prototype._addAction = function (action) {
-  if (action.isOptional()) {
+  if (action.isRequired()) {
```

Comment:
https://github.com/python/cpython/blob/2bd37c227e8a042e036c7455d974e3d0b36aed53/Lib/argparse.py#L1725

also don't see anything similar in mainstream